openmdao/core/problem.py

```diff
@@ -103,6 +103,8 @@ def __init__(self, model=None, comm=None, use_ref_vector=True, root=None):
 
         self._mode = None  # mode is assigned in setup()
 
+        recording_iteration_stack = []
+
```

Comment:
I could try it on the OpenMDAO cluster, right? Will take me a while to setup the environment there.